less/plugins/adapt-contrib-hotgraphic/hotgraphic.less

```diff
@@ -26,4 +26,39 @@
     text-align: center;
   }
   // --------------------------------------------------
+  // _useGraphicsAsPins config
+  // --------------------------------------------------
+  // _use
+  &__tile-item {
+    position: relative;
+    z-index: 4;
+
+    .hotgraphic__tile {
+      &:before {
+        content: '';
+        position: absolute;
+        top: 0px;
+        left: 0px;
+        width: 100%;
+        height: 100%;
+        background: fadeout(@item-color, 100%);
+        transition: background @duration ease-in;
+      } 
+    }
+
+    .no-touch &:hover {
+      z-index: 5;
+      
+      .hotgraphic__tile {
```

Comment:
Suggest removing the nesting, utilising the inbuilt transform mixin, and using multiple ampersand symbols.

```
.no-touch &__tile-item:hover &__tile {
  box-shadow: 0 2px 5px fade(@black, 20%);
  .transform(scale(1.03));
  .transition(transform @duration ease-in, box-shadow @duration ease-in;);
}
```